ecosystem/sf-indexer/aptos-sf-indexer/src/counters.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_metrics_core::{
+    register_int_counter, register_int_counter_vec, IntCounter, IntCounterVec,
+};
+use once_cell::sync::Lazy;
+
+/// Number of times a given processor has been invoked
+pub static PROCESSOR_INVOCATIONS: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "indexer_processor_invocation_count",
+        "Number of times a given processor has been invoked",
+        &["processor_name"]
+    )
+    .unwrap()
+});
+
+/// Number of times any given processor has raised an error
+pub static PROCESSOR_ERRORS: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "indexer_processor_error_count",
+        "Number of times any given processor has raised an error",
```

Comment:
`a given`

Reply:
Ha, you are your own best grammar police.